### PR TITLE
v3.1.0-rc1 Release

### PR DESCRIPTION
**Pull request by webron**
Changelog
---------

See [3.1.0-rc0](https://github.com/OAI/OpenAPI-Specification/releases/tag/3.1.0-rc0) for previous changes in 3.1.0, including the explanation of why there are breaking changes.

### Breaking changes
- *Server Variable*'s `enum` now MUST not be empty (changed from SHOULD).
- *Server Variable*'s `default` now MUST exist in the `enum` values, if such values are defined (changed from SHOULD).
- `responses` are no longer required to be defined under the *Operation Object*.

### Clarifications

- It is now clarified when path template expression may not have a corresponding path parameter.
- Data types (and just primitive data types) now correspond to JSON Schema.
- Various cosmetic fixes.
- A new section was added to address how to handle the `$schema` keyword (implicitly and explicitly).

### CODE REVIEW

**Review by darrelmiller — APPROVED** [review]

File: versions/3.1.0.md [hunk 1]
```diff
@@ -2336,6 +2331,14 @@ As such, inline schema definitions, which do not have a given id, *cannot* be us
 The [xml](#schemaXml) property allows extra definitions when translating the JSON definition to XML.
 The [XML Object](#xmlObject) contains additional information about the available options.
 
+###### Picking Schema Vocabularies
+
+It is important for tooling to be able to detect what meta-schema any given resource wishes to be processed with: JSON Schema Core, JSON Schema Validation, OpenAPI Schema Object, or some custom meta schema.
+
+`$schema` MAY be present in any Schema Object, and if present MUST be used to determine which dialect should be used when processing the schema.
+
+When `$schema` is not present, the default the following dialect MUST be assumed: `$schema: "https://spec.openapis.org/oas/3.1/schema-object"`.
```
Comment (adamaltman) [inline versions/3.1.0.md:2340]: Is something supposed to exist at this URL "https://spec.openapis.org/oas/3.1/schema-object"? (I get a 404, and I expected a schema object.)

Comment (adamaltman) [reply versions/3.1.0.md:2340]: I opened this issue: https://github.com/OAI/OpenAPI-Specification/issues/2371 about this comment.

File: README.md [hunk 2]
```diff
@@ -19,9 +19,9 @@ This GitHub project is the starting point for OpenAPI. Here you will find the in
 
 The current version of the OpenAPI specification is [OpenAPI Specification 3.0.3](versions/3.0.3.md).
```
Comment (JelisaCooper) [inline README.md:20]: Original Auto Salez

**Review by JelisaCooper — COMMENTED** [review]
Original

**Pull request merged**